### ISSUE TITLE
Create metadata dictionary in version 6

### ISSUE THREAD

**Issue by marcuslindblom** [ravendb/ravendb-nodejs-client#460]
In an older project running ravendb 5.4 I import `MetadataDictionary` like this and use at like the example below.
```ts
import { MetadataDictionary } from 'ravendb/dist/Mapping/MetadataAsDictionary';
for (const template of templates.filter((t) => !t.disableURL)) {
    await bulkInsert.store(template, `templates/${template.collection}`, createMetadataDictionary({ '@collection': 'Templates' }));
}
```

But when running ravendb 6 it does not exist, so how can I do this in the latest version?

**Comment by ml054** [comment]
@marcuslindblom Looks like we didn't expose MetadataAsDictionary from index.ts in package. 

I'm reporting this as an issue. 
As temporary workaround you can use:
```
                    await bulkInsert.store(new Company(), "test/" + i, {
                        isDirty(): boolean {
                            return false;
                        },
                        "@collection": "Test1"
                    })
```

**Comment by ml054** [comment]
https://issues.hibernatingrhinos.com/issue/RDBC-875/Expose-MetadataDictionary-in-index.ts

**Comment by marcuslindblom** [comment]
Thanks!

When using nodejs and typescript, should I add something similar to `Raven-Clr-Type`?

**Comment by ml054** [comment]
If you are creating new project you can use ClassTransformer which was introduced in v6.0. 

It is listed here:
https://github.com/ravendb/ravendb-nodejs-client/releases/tag/6.0.0
and detailed link to example:
https://github.com/ravendb/ravendb-nodejs-client/blob/5404b45c83d7d55a0a8754e5984b6e9685f1ec6f/test/Ported/CustomSerializationTest.ts#L16

Then we won't need nested fields types (to keep info that field is date). 

And you don't need Raven-Clr-Type as well.

**Comment by ml054** [comment]
Fixed in #463
